### DISCUSSION TITLE
Is it possible to start Minio app with 2 storage locations?

### DISCUSSION THREAD

**Discussion by SlavikCA** [minio/minio#11517] [Q&A]
I'm not sure, what's the correct terminology, so please correct me, if I don't use right term.

Currently, I'm using Minio without any erasure codes, with one location. It's started like this:
```
server /mnt/ssd
```

Now, I added another drive to OS and want to have some buckets on new drive. How do I do that?
I tried this:
```
server /mnt/ssd /mnt/hdd
```
But Minio fails to start like this.

**Comment by klauspost** [comment]
To run MinIO in distributed mode, you must add 4 disks, as the error message probably will tell you.

See https://docs.min.io/minio/baremetal/concepts/erasure-coding.html

**Reply by SlavikCA** [reply]
Yes, that's what error message is telling.

No, I'm not looking to run in distributed mode. I'm looking for single-node service running on 2 drives (discs)

**Reply by klauspost** [reply]
@SlavikCA Distributed can be single-node, but requires 4 disks.

**Reply by SlavikCA** [reply]
That's right. But again: I'm not looking to run in distributed mode. I'm trying to find a way to run single node NOT in distributed mode with two discs.

**Reply by klauspost** [reply]
There is nothing stopping you from making a RAID 0/1 of your disks. You will have to set that up outside MinIO though.
